test/test_st.t

```diff
@@ -151,17 +178,18 @@ Tests Model Tree:
   Execution mode: st
   
   >Program Print
-  (Val Int 6)
+  6
   
   >Program Print
-  (SymbInt $_descending)
+  $_descending
   
   >Program Print
-  (+ (+ (SymbInt $_descending) (Val Int 3)) (* (Val Int 5) (Val Int 2)))
+  (int.add (int.add $_descending 3) 10)
   
   >Program Print
-  (+ (+ (+ (SymbInt $_descending) (Val Int 3)) (* (Val Int 5) (Val Int 2))) (Val Int 1))
+  (int.add (int.add (int.add $_descending 3) 10) 1)
   
+  Returned (int.add (int.add $_descending 3) 10)
```

Comment:
We should create an issue on encoding due to incomplete simplifications because this should be:

```
(int.add (int.add $_descending 3) 10) = (int.add $_descending 13)
(int.add (int.add (int.add $_descending 3) 10) 1)  = (int.add $_descending 14)
```